ruby/example_code/s3/tests/test_s3_get_csaes_decrypt_item.rb

```diff
@@ -0,0 +1,82 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX - License - Identifier: Apache - 2.0
+
+require_relative '../s3_get_csaes_decrypt_item'
+
+describe '#get_decrypted_object_content' do
+  # Captures the data (metadata and body) put to an Amason S3 object.
+  def stub_put(s3_client)
+    data = {}
+    s3_client.stub_responses(:put_object, lambda { |context|
+      data[:metadata] = context.params[:metadata]
+      data[:enc_body] = context.params[:body].read
+      {}
+    })
+    data
+  end
+
+  # Given data from stub_put, stub a get for the same object
+  # during get get_object is called twice, once to get the full body and
```

Comment:
Are lines 18 & 19 separate comments? If 2, then
# Given data... object
# During get, get_object is called...

If one comment, then should this be
# Given data...for the same object during get, get_object is called...